internal/deviations/deviations.go

```diff
@@ -200,4 +200,7 @@ var (
 
 	SkipBGPTestPasswordMismatch = flag.Bool("deviation_skip_bgp_test_password_mismatch", false,
 		"Skip BGP TestPassword mismatch subtest if value is true, Default value is false")
+
+	NoBGPNetworkInstanceTable = flag.Bool("deviation_no_bgp_network_instance_table", false,
```

Comment:
@trathod1 is it better to name it as: `NetworkInstanceTableDeletionNotRequired`? (Because it is about deletion).

Reply:
@xw-g Current operation is specific to deletion. Considering, it is always going to be for deletion, sure I can name it as you mentioned. I was earlier thinking to keeping it based on entity (if table exists or not) instead of operation (config/delete/update).